.pre-commit-config.yaml

```diff
@@ -5,10 +5,6 @@ repos:
     rev: 22.6.0
     hooks:
       - id: black
-  - repo: https://github.com/pycqa/isort
```

Comment:
Why do you remove the isort check?

Reply:
Sorry, I didn't read the PR description. 

I can not reproduce the issue with pre-commit. Let's talk in private.

Reply:
update the version